frc_robot_hw/src/frc_robot_hw_real.cpp

```diff
@@ -490,6 +494,46 @@ bool FRCRobotHWReal::init(ros::NodeHandle& root_nh, ros::NodeHandle& robot_hw_nh
   return true;
 }
 
+void FRCRobotHWReal::doSwitch(const std::list<hardware_interface::ControllerInfo>& start_list,
+                              const std::list<hardware_interface::ControllerInfo>& stop_list) {
+
+  FRCRobotHW::doSwitch(start_list, stop_list);
+
+  using Mode = MultiPIDController::Mode;
+
+  // Disable PID for simple speed controllers claimed by stopping controllers
+  for (const auto& controller : stop_list) {
+    for (const auto& claimed : controller.claimed_resources) {
+      for (const auto& resource : claimed.resources) {
+        if (simple_speed_controller_pids_.find(resource) == simple_speed_controller_pids_.end())
```

Comment:
why not
```
if (... != ...) {
  ...
}
```
instead of `continue`?

Reply:
Removes a level of indentation in an already very indented block

Reply:
alright :+1:

Reply:
In this case I suppose it's not a big deal, but in the starting controllers block it makes a difference, so I repeated it here for consistency